src/App.jsx

```diff
@@ -298,10 +359,14 @@ function KanbanBoard() {
                                   onClick={() => {
                                     const taskContent = prompt('Enter task content:');
                                     if (taskContent?.trim()) {
+                                      const dueDate = new Date();
+                                      dueDate.setDate(dueDate.getDate() + 7); // Default to 1 week from now
```

Comment:
![kody code-review](https://img.shields.io/badge/kody-code--review-brightgreen)

```suggestion
const dueDate = await showDatePickerDialog() || new Date(Date.now() + 7 * 24 * 60 * 60 * 1000);
```
The addTask function uses a hardcoded 7-day due date. Consider making this configurable or allowing users to select the due date when creating a task.



<sub>Talk to Kody by mentioning @kody</sub>

<sub>Was this suggestion helpful? React with 👍 or 👎 to help Kody learn from this interaction.</sub><!-- kody-codereview -->&#8203;
&#8203;